docs/planning/index.markdown

```diff
@@ -21,9 +21,12 @@ Full real-time data sync where updates are initiated from both the client and se
 
 ### AeroGear Push - 1.1.0 (April 2015)
 
-Next version of the Unified Push Server, containing new features like support for new Push networks or advanced analytis.
+Next version of the Unified Push Server, containing new features like support for new Push networks or advanced analytis:
```

Comment:
:-) fixed the typo

On Tue, Jan 20, 2015 at 9:11 AM, Sebastien Blanc <notifications@github.com>
wrote:

> In docs/planning/index.markdown
> <https://github.com/aerogear/aerogear.org/pull/467#discussion-diff-23207471>
> :
>
> > @@ -21,9 +21,12 @@ Full real-time data sync where updates are initiated from both the client and se
> >
> >  ### AeroGear Push - 1.1.0 (April 2015)
> >
> > -Next version of the Unified Push Server, containing new features like support for new Push networks or advanced analytis.
> > +Next version of the Unified Push Server, containing new features like support for new Push networks or advanced analytis:
>
> analytics ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/467/files#r23207471>.
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf